### ISSUE TITLE
Upstream our changes to PyTorch r(2+1)d architecture

### ISSUE THREAD

**Issue by daniel-j-h** [moabitcoin/ig65m-pytorch#5]
The torchvision r(2+1)d architecture needs two modifications to get it in sync with the official Caffe2 implementation (see https://github.com/facebookresearch/VMZ/issues/89) and our provided code:
- Number of midplanes in the downsampling blocks
- Batchnorm

We should upstream both modifications to torchvision.

**Comment by bjuncek** [comment]
see https://github.com/pytorch/vision/issues/1265

need some discussion, but is easily done

**Comment by daniel-j-h** [comment]
@bjuncek great, this would be sweet to have upstream! :heart: Please consider adapting the Batchnorm blocks, too. Then the architectures are 100% in sync and weights can easily be transfered.

**Comment by daniel-j-h** [comment]
Closing this ticket since there is nothing actionable on our end. We will follow up with you folks on vmz to upstream some of our changes when we ported the weights.
